src/pages/index.astro

```diff
@@ -4,44 +4,51 @@ import "../index.scss";
 import CardProject from "../components/CardProject.astro";
 
 const weatherCard = {
-  imgSrc: "astro-lp/project_1.png",
-  imgAlt: "Weather App Layout",
+  img: {
+    src: "astro-lp/project_1.png",
+    alt: "Weather App Layout",
+  },
   title: "Weather App",
   description:
     "Stay ahead of the forecast with our intuitive weather app. Get real-time updates on current conditions, hourly forecasts, and daily outlooks. Plan your day confidently with accurate weather information at your fingertips. Never be caught off guard by the elements again!",
   skills: ["HTML", "CSS", "JavaScript"],
```

Comment:
This value is the same for every card, lets move it to `const baseSkills = ["HTML", "CSS", "JavaScript"];` and assign this const to each obj

Reply:
Changed